app/assets/stylesheets/aspects.scss

```diff
@@ -4,20 +4,26 @@
     @include selectable-list;
 
     .status_indicator {
-      width: 19px;
       height: 14px;
+      width: 19px;
       display: inline-block;
     }
 
     a {
       .text {
-        color: #333333;
+        color: #333;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.

Reply:
This line is not related with this PR, so I would just leave it as it is.